src/core/server/http/router/validator/validator.ts

```diff
@@ -143,8 +143,8 @@ export type RouteValidatorFullConfig<P, Q, B> = RouteValidatorConfig<P, Q, B> &
  * @internal
  */
 export class RouteValidator<P = {}, Q = {}, B = {}> {
-  public static from<P = {}, Q = {}, B = {}>(
-    opts: RouteValidator<P, Q, B> | RouteValidatorFullConfig<P, Q, B>
+  public static from<_P = {}, _Q = {}, _B = {}>(
+    opts: RouteValidator<_P, _Q, _B> | RouteValidatorFullConfig<_P, _Q, _B>
```

Comment:
Is that really an opinionated decision from the `@typescript-eslint/no-shadow` rules? There are technically no shadowing between the class/instance generics and a static function's...